parser-typechecker/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -1097,7 +1102,7 @@ renderEditConflicts ppe Patch{..} =
       P.oxfordCommas [ termName r | TermEdit.Replace r _ <- es ]
     formatConflict = either formatTypeEdits formatTermEdits
 
-type Numbered a = State.State Int a
+type Numbered a = State.State (Int, Seq.Seq String) a
```

Comment:
Would've been more self-documenting if this was `NumberedArgs` again, but I wanted O(1) snoc and was lazy to patch all the `[String]` <-> `Seq String`.  idk

Reply:
Seems fine.